pkg/ovs/openflow/ofctrl_nxfields.go

```diff
@@ -82,7 +82,7 @@ func (m *CtMark) GetRange() *Range {
 }
 
 func (m *CtMark) GetValue() uint32 {
```

Comment:
What about adding a new function like `GetCTValue()` to return the value in the conntrack, and leave the current function here to retrieve the original value set in the CtMark?

Reply:
Got it. I added a new function called `GetValueWithOffset()` to get the value with offset and left the function `GetValue()` to return the original value set in the CtMark.